src/components/ui/DataGrid/DataGridActions.svelte

```diff
@@ -20,7 +21,7 @@
     placement: Placement;
   };
 
-  export let rowData: RowData | undefined;
+  export let cancellable: boolean = true;
```

Comment:
Can we rename this to explicitly tie it to downloading? e.g. `isDownloadCancellable` or something like that?